common/djangoapps/third_party_auth/management/commands/saml.py

```diff
@@ -107,52 +111,75 @@ def _check_provider_configurations(self):
 
         for provider_config in provider_configs:
             total_providers += 1
+
+            # Check if provider is disabled
+            provider_disabled = not provider_config.enabled
+            disabled_status = "[DISABLED] " if provider_disabled else ""
+
             provider_info = (
-                f"Provider (id={provider_config.id}, name={provider_config.name}, "
-                f"slug={provider_config.slug}, site_id={provider_config.site_id})"
+                f"{disabled_status}Provider (id={provider_config.id}, "
+                f"name={provider_config.name}, slug={provider_config.slug}, "
+                f"site_id={provider_config.site_id})"
```

Comment:
[optional] I'm wondering about a possible different format:
```
disabled_status = ", enabled=False" if provider_disabled else ""
```
And then:
```suggestion
                f"Provider (id={provider_config.id}, "
                f"name={provider_config.name}, slug={provider_config.slug}, "
                f"site_id={provider_config.site_id}{disabled_status})"
```
The same could be used for disabled saml configs. 

Which do you prefer?

Reply:
I prefer the suggested format with [enabled=False] at the end because
 representation patterns like `Provider(id=1, name=test, enabled=False)`


as that DISABLED prefix was more of a visual flag